### PR TITLE
Bug 1129822 - Print node/npm versions

### PR DESCRIPTION
**Pull request by timdream**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[timdream Timothy Guan-tin Chien (timdream)](https://github.com/timdream) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5cdfbbd0f21d)

File: Makefile [hunk 1]
```diff
@@ -719,6 +719,8 @@ else
 	rm -fr node_modules
 	cp -R $(NODE_MODULES_SRC)/node_modules node_modules
 endif
+  node --version
```
Comment (RickyChien) [inline Makefile]: we should use $(NODEJS) and $(NPM) which are already defined at [1].

[1] https://github.com/timdream/gaia/blob/node-version/Makefile#L465-L471

File: Makefile [hunk 2]
```diff
@@ -719,6 +719,8 @@ else
...
+  npm --version
```
Comment (RickyChien) [inline Makefile]: Please fix the indent

**Comment by try-server-hook** [comment]
[timdream Timothy Guan-tin Chien (timdream)](https://github.com/timdream) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2c726b680bb4)

**Pull request merged**